libs/ha/ha_common_libs_k8s.py

```diff
@@ -1855,3 +1856,87 @@ def dnld_obj_verify_chcksm(self, s3_test_obj, bucket, obj, download_path, upload
             return False, download_checksum
         LOGGER.info("Successfully downloaded the object and verified the checksum")
         return True, download_checksum
+
+    def object_overwrite_dnld(self, s3_test_obj, s3_data, iteration, random_size, event=None,
+                              queue=None, background=False):
+        """
+        Function to create bucket, put object and overwrite existing object
+        :param s3_test_obj: Object of s3 test lib
+        :param s3_data: Dict that contains bucket and object info
+        ({self.bucket_name: [self.object_name, file_size]})
+        :param iteration: Number of iterations
+        :param random_size: Flag to select random size of object
+        :param event: Event for background IOs
+        :param queue: Queue to fill output in case of background IOs
+        :param background: Flag to start background process
+        :return: (bool, dict)
+        """
+        checksums = dict()
+        fail_count = 0
+        test_dir_path = os.path.join(TEST_DATA_FOLDER, "HAOverWrite")
+        if not os.path.isdir(test_dir_path):
```

Comment:
do we want to rather remove if any dir exists and create new one? Or is it because same file path will be needed next time?

Reply:
Same dir will be used for all overwrite operations. No need to remove if dir already exists.